docs/client-sdk/javascript/tutorials/start-messaging.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: Start messaging with XMTP
+sidebar_position: 1
+---
+
+# Start messaging with XMTP
+
+XMTP is an open protocol and network for secure web3 messaging. 
+
+Developers build with [XMTP client SDKs](/sdks-and-tools) to provide messaging in their apps. The XMTP messaging API client takes care of:
+
+- [Authentication](/docs/dev-concepts/account-signatures) using an **XMTP identity that you own and control**
+- [End-to-end encryption](/docs/dev-concepts/invitation-and-message-encryption) of **messages that you own and control**
+- Providing an **[interoperable inbox](/docs/dev-concepts/interoperable-inbox)** accessible across apps built with XMTP
+- Relaying messages to the **progressively decentralized** [XMTP network](/docs/dev-concepts/architectural-overview#network-layer)
+
+A great way to start learning how to build an app with XMTP is to use one!
+
+**To start messaging with XMTP:**
+
+1. **Pick an identity** you want to use to message with XMTP.
+
+    You can use an Ethereum address to start.
+    
+    If you have an [ENS name](https://ens.domains/) or [Lens handle](https://claim.lens.xyz/), you can use it instead. These human-readable web3 identities make it easier for you to share your address so others can message you.
+
+    Don’t have an Ethereum address? [Create a MetaMask wallet](https://metamask.io/) to get one.
+    
+2. **Pick an app.**
+
+   - [Converse](https://getconverse.app/) (iOS): Works with Ethereum addresses, ENS names, and Lens handles
+   - [XMTP Inbox](https://xmtp.chat/) (web): Works with Ethereum addresses and ENS names
+   - [Buttrfly](https://buttrfly.app/) (iOS): Works with Lens handles
+   - [Lenster](https://lenster.xyz/) (web): Works with Lens handles
+   - [Orb](https://orb.ac/) (ios and Android): Works with Lens handles
+
```

Comment:
Did I capture the web3 identity compatibility per app correctly above?

Reply:
I think so. It may be difficult to keep track of which app supports which resolution over time though. If that happens, we could remove the identity compat info

For the send a message section, can also add gm.xmtp.eth bot and the 0x address for hi.xmtp.eth? It'll be helpful for users to 

1. Experience the delight of receiving a response (gmbot)
2. Test messaging 0x addresses as well as ENS names